<?xml version="1.0"?>
<launch>
	
  <!-- start object detector -->
  <node name="dpm_detect_service" pkg="ffld_ros" type="dpm_detect_service" output="screen">
    <param name="model_path" value="$(find ffld_nx)/data/models/"/>
    <rosparam file="$(find ffld_ros)/include/models.yaml"/>
  </node>
  
  <!-- send it an image -->
  <node name="object_detector_client" pkg="ffld_ros" type="detect_object_client"
        output="screen" args="$(find ffld_nx)/data/images/chair_trainset_new/pic15.jpg"/>

</launch>
